scripts/ingress-router.sh

```diff
@@ -43,13 +49,35 @@ list_manifests_reversed() {
   list_manifests | tac
 }
 
+add_host() {
+  local host context
+  context=$(kubectl config current-context)
+  host=$(kubectl get routes -l 'app.kubernetes.io/managed-by=rhacs-fleetshard' --all-namespaces -o json | jq -r '.items[].spec.host' | fzf --header "Using context $context")
+  sudo hostctl add domains "$HOSTCTL_PROFILE" "$host"
+}
+
+remove_host() {
+  local host
+  host=$(hostctl list "$HOSTCTL_PROFILE" -o json | jq -r ".[].Host" | fzf)
```

Comment:
```suggestion
  host=$(hostctl list "$HOSTCTL_PROFILE" -o json | jq -r ".[].Host" | fzf --header "Using context $context")
```

Or something similar?

Reply:
This does not call `kubectl` just prints the list from `/etc/hosts`. I could add a profile here.